solcjs

```diff
@@ -11,48 +11,29 @@ var smtsolver = require('./smtsolver.js');
 // FIXME: remove annoying exception catcher of Emscripten
 //        see https://github.com/chriseth/browser-solidity/issues/167
 process.removeAllListeners('uncaughtException');
-
-var yargs = require('yargs')
-  .usage('Usage: $0 [options] [input_file...]')
-  .option('version', {
-    describe: 'Show version and exit.',
-    type: 'boolean'
-  })
-  .option('optimize', {
-    describe: 'Enable bytecode optimizer.',
-    type: 'boolean'
-  })
-  .option('bin', {
-    describe: 'Binary of the contracts in hex.',
-    type: 'boolean'
-  })
-  .option('abi', {
-    describe: 'ABI of the contracts.',
-    type: 'boolean'
-  })
-  .option('standard-json', {
-    describe: 'Turn on Standard JSON Input / Output mode.',
-    type: 'boolean'
-  })
-  .option('output-dir', {
-    alias: 'o',
-    describe: 'Output directory for the contracts.',
-    type: 'string'
-  })
-  .version(solc.version())
-  .showHelpOnFail(false, 'Specify --help for available options')
-  .help()
-
-var argv = yargs.argv;
-var files = argv._;
-var destination = argv['output-dir'] || '.'
+var commander = require('commander');
+
+var program = new commander.Command();
+program.name('solcjs');
+program.version(solc.version());
+program
+  .option('--version', 'Show version and exit.')
+  .option('--optimize', 'Enable bytecode optimizer.')
+  .option('--bin', 'Binary of the contracts in hex.')
+  .option('--abi', 'ABI of the contracts.')
+  .option('--standard-json', 'Standard JSON configuration file.')
```

Comment:
This doesn't take a file name so that the help text is confusing. I'd suggest to stick to the original text.

Reply:
Oops, I have no idea why I changed that.  Fixed.

Reply:
Ugh, I pushed the wrong branch... give me a few minutes while I clean up the mess I just created for myself.

Reply:
Fixed.